docs/triggertemplates.md

```diff
@@ -64,6 +64,9 @@ One instance where there is useful is when resources in a `TriggerTemplate` have
 When creating resource, an additional label is added on created resources with the following value: `event-listener-name`: `<EventListenerName>`.
 Mark all resources created by a certain EventListener, which helps with housekeeping.
 
+When creating resource, another additional label is added on created resources with the following value: `triggers-eventid`: `<EventID>`.  The `<EventID>` is assigned for each event processed by the EventListener. 
```

Comment:
Can we update the key here to be `tekton.dev/triggers-eventid`? Also, the above key should be `tekton.dev/eventlistener`.

Reply:
I make changes